test/choosetests.jl

```diff
@@ -127,9 +127,6 @@ function choosetests(choices = [])
                    "linalg/generic", "linalg/uniformscaling", "linalg/lq",
                    "linalg/hessenberg", "linalg/rowvector", "linalg/conjarray",
                    "linalg/blas"]
-    if Base.USE_GPL_LIBS
```

Comment:
This is confusing to me. ARPACK is not GPL-licensed, so why did we only test the Arnoldi stuff if we were using GPL libraries?

Reply:
Guessing: maybe because it depends on SuiteSparse which is partly GPL?

Reply:
That would make sense. I guess I should modify the stdlib tests then to only run Arnoldi when we're okay with using GPL libraries?

Reply:
@ViralBShah ^

Reply:
So ARPACK as such doesn't depend on anything in SuiteSparse but the wrappers use factorizations and when the input is sparse, which is the case in some of the tests, then the wrappers use sparse factorizations from SuiteSparse.

Reply:
Yes, the shift and invert solvers need SuiteSparse.